src/components/NotificationModal/index.tsx

```diff
@@ -219,20 +235,25 @@ export const NotificationModal: React.FC<NotificationModalProps> = ({
     </Modal>
   );
 };
+
 export interface SubscriptionMethodControl {
   label: string;
   placeholder: string;
   targetPath: string;
   validator: (v: string) => boolean;
 }
 
+export type ResourceType = "TASK" | "BUILD";
+
+const inputWidth = "width: calc(80% - 50px);";
```

Comment:
Just curious What is this 50px accounting for?

Reply:
https://github.com/evergreen-ci/spruce/pull/303#discussion_r456641757